test/resolvers/project-version.js

```diff
@@ -111,6 +117,48 @@ describe('ProjectVersion resolver', () => {
         });
     });
 
+    describe('Additional availability', () => {
+      it('creates new ProjectEstablishment models for additional establishments', () => {
+        const establishments = [
+          {
+            'establishment-id': 8202
+          }
+        ];
+        const opts = {
+          action: 'patch',
+          id: versionId,
+          data: {
+            patch: jsondiff.diff({}, { establishments })
+          }
+        };
+        return Promise.resolve()
+          .then(() => this.projectVersion(opts))
+          .then(() => this.models.ProjectEstablishment.query().where({ projectId, establishmentId: 8202 }).first())
+          .then(projectEstablishment => {
+            assert.equal(projectEstablishment.establishmentId, 8202);
+            assert.equal(projectEstablishment.projectId, projectId);
+            assert.equal(projectEstablishment.status, 'draft');
```

Comment:
the first 2 assertions are just testing objection rather than your code, so they're probably redundant?

Maybe assert against how many relations are created?

Reply:
fair